### PR TITLE
Fixes #20374: Test ncf in docker

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/20374

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

File: Jenkinsfile [hunk 1]
```diff
@@ -23,57 +71,47 @@ pipeline {
...
-                script {
-                    deleteDir()
-                    dir('ncf') {
-                        checkout scm
-                        sh script: 'git log -1 --pretty=%B'
+            matrix {
+                axes {
+                    axis {
+                        name 'OS'
+                        values 'debian:11'
```
Comment (Fdall) [inline Jenkinsfile:87]: Dockerfile is a ubuntu20 based one, not debian11

Comment (Fdall) [reply Jenkinsfile:87]: My bad

**Review by Fdall — APPROVED** [review]

**Pull request merged**